File: model/src/main/java/net/artux/pda/model/items/ArmorModel.java
package net.artux.pda.model.items;

import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = true)
public class ArmorModel extends WearableModel {

    private float thermalProtection;
    private float electricProtection;
    private float chemicalProtection;
    private float radioProtection;
    private float psyProtection;
    private float damageProtection;
    private float condition;

}

File: model/src/main/java/net/artux/pda/model/items/ArmorModel.kt
package net.artux.pda.model.items

data class ArmorModel(
    var thermalProtection: Float = 0f,
    var electricProtection: Float = 0f,
    var chemicalProtection: Float = 0f,
    var radioProtection: Float = 0f,
    var psyProtection: Float = 0f,
    var damageProtection: Float = 0f,
    var condition: Float = 0f
) : WearableModel()